lib/components/viewers/RouteRow.js

```diff
@@ -14,20 +16,31 @@ import { getFormattedMode } from '../../util/i18n'
 import DefaultRouteRenderer from '../narrative/metro/default-route-renderer'
 import OperatorLogo from '../util/operator-logo'
 
-import RouteDetails from './route-details'
-
 export const StyledRouteRow = styled.li`
+  align-items: center;
   background-color: white;
-  margin: 10px 0;
+  display: flex;
+  gap: 7px;
+  justify-content: space-between;
   list-style: none;
+  margin: 5px 0;
+  padding: 5px;
+  position: relative;
 `
-
+// Route Row Button sits invisible on top of the route name and info.
 export const RouteRowButton = styled(Button)`
-  align-items: center;
-  display: flex;
-  padding: 6px;
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
   transition: all ease-in-out 0.1s;
   width: 100%;
+
+  &:active,
+  &.btn-default:active:focus {
+    background-color: rgba(212, 212, 212, 0.2);
```

Comment:
In Bootstrap there's a default `active:focus` state that makes the background color grey, but since now the button is an overlay, it's turning everything grey! I tried to just lower the opacity, but I agree it looks weird, I'm just going to take the whole thing out!